audtorch/datasets/speech_commands.py

```diff
@@ -0,0 +1,115 @@
+import os
+import random
+
+from .utils import (download_url, extract_archive, safe_path)
+from .base import AudioDataset
+from .utils import safe_path
+
+
```

Comment:
In order to make the `doctest` not fail it would be the easiest to include something like
```python
__doctest_skip__ = ['*']
```
here, compare e.g. https://github.com/audeering/audtorch/blob/master/audtorch/datasets/libri_speech.py

It's not ideal, but so far the easiest approach for the data sets where testing without downloading the actual data is not that straight forward.

Reply:
Ah I see, makes sense. Thanks!